hoomd/System.cc

```diff
@@ -167,9 +167,9 @@ void System::run(uint64_t nsteps, bool write_at_start)
             if ((*analyzer->getTrigger())(m_cur_tstep))
                 analyzer->analyze(m_cur_tstep);
             }
-        }
 
-    updateTPS();
+        updateTPS();
```

Comment:
This was working at one point: https://hoomd-blue.readthedocs.io/en/v3.0.0/tutorial/02-Logging/04-Writing-Formatted-Output.html

It was broken in #1264 : https://github.com/glotzerlab/hoomd-blue/pull/1264/files#diff-a241487071270e42a76be6ecb1d44bfeb8a0b5512de965d893dfd8517db0e347

Reply:
Yep, I noticed that I messed it up.